packages/terra-content-container/tests/wdio/content-container-spec.js

```diff
@@ -0,0 +1,18 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('medium');
+
+describe('Content Container', () => {
```

Comment:
Lets add
```before(() => browser.setViewportSize(Terra.viewports('medium')[0]));```
and drop each specification of `veiwports` in the test steps
so,
```diff
+    Terra.should.beAccessible();
-    Terra.should.beAccessible({ viewports });
```

Reply:
Updated: https://github.com/cerner/terra-core/pull/1468/commits/9790d7179668344164cbceea5bcf83e822169d6a